examples/examples_go_test.go

```diff
@@ -18,19 +18,19 @@ package examples
 
 import (
 	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
-	// "path/filepath"
+	"path/filepath"
 	"testing"
 )
 
-// func TestAccTrail(t *testing.T) {
```

Comment:
Strange it was commented out, there's a surgical thing to do, `t.Skipf("This test is skipped because...")`

Reply:
Oh that was just me, not thinking of `Skipf`. Thanks for the reminder.